Dockerfile

```diff
@@ -1,12 +1,56 @@
-FROM jpetazzo/dind
+FROM jpetazzo/dind AS builder
```

Comment:
use this image as base for builder: `iofog/ubuntu-16.04-java8 `

Reply:
updated